mochimochi/classifier/binary/pa.hpp

```diff
@@ -34,23 +35,27 @@ public :
 
     // int select : switching the PA algorithm
     // 0 : PA
-    // 1 : PA-1
-    // 2 : PA-2
+    // 1 : PA-I
+    // 2 : PA-II
     switch(kSelect) {
     case 0 :
-      _compute_tau = [](const auto value, const auto loss) {
-        return loss / std::pow(std::abs(value), 2);
+      _compute_tau = [=](const auto value, const auto loss) {
+        /* Check for divide by zero in which case return zero for tau. */
+        /* If "value" is non-zero then proceed with division and return result. */
```

Comment:
So if I understand the NaN was coming from the Inf being serialized?

Reply:
The tau value is used to calculate something else and plugging in inf to that calculation resulted in nans.